src/sentry/lang/native/processing.py

```diff
@@ -283,6 +290,32 @@ def _handles_frame(data, frame):
     return is_native_platform(platform) and "instruction_addr" in frame
 
 
+def _expand_addr_mode(addr_mode, modules):
+    if addr_mode in (None, "abs"):
+        return "abs"
+    if addr_mode.startswith("rel:"):
+        arg = addr_mode[4:]
+
+        for idx, module in enumerate(modules):
```

Comment:
Before iterating modules, you can use symbolic to check if this is even a debug id. Alternatively, what if we try the number first, and only then check debug_ids which is more "expensive".